lib/fml_forms/validations.rb

```diff
@@ -10,6 +10,7 @@ module FML
   #   Given a spec like:
   #    form:
   #      title: "Validation Example"
+  #      id: 0acd2dc0b8b325eb5698e50387b79d00d77b1f10
```

Comment:
I would *much* prefer to generate them programmatically, but I don't see how, if you have ideas I'm all ears